core/src/test/resources/ssl-testing/generate_ssl_test_credentials.sh

```diff
@@ -16,6 +16,8 @@ echo "00" > cert.srl
 openssl genrsa -f4 2048 > server_key.pem
 openssl rsa -in server_key.pem -pubout > server_pubkey.pem
 openssl req -key server_key.pem -new -out server.req -subj "/CN=$server_name"
+** convert from SSLeay to pkcs8
```

Comment:
SSLeay is the default format for openssl private keys. In their infinite NiH, java doesn't easily read them.